layers/+lang/python/config.el

```diff
@@ -28,3 +28,6 @@
   "Automatically set pyenv version from \".python-version\".
 
 Possible values are `on-visit', `on-project-switch' or `nil'.")
+
+(add-hook 'python-mode-hook
+          (lambda () (set (make-local-variable 'comment-inline-offset) 2)))
```

Comment:
1. Can use `setq-local` here
2. Why not set the variable inside of `python-default` instead? (see `python/init-python`in packages.el)